Select_GCMs_Centroid.Rmd

```diff
@@ -117,7 +121,7 @@ historic_all <- historic_dfs[[1]] %>%
 # Determine low-skill models using list created from Rupp et al. 2016 (this is from the CCRP team)
 
 # assign region of park (one of SWR, SER, PWR or 'mean' if none of these)
-region <- "SWR"
+region <- "SER"
 
 
 low_skill_models <-
```

Comment:
could do the same thing here as you do in select_GCMs: `read_delim(
        "https://raw.githubusercontent.com/rossyndicate/CCRP_automated_climate_futures/master/data/general/GCM_skill_by_region.txt"
      )`